README.md

```diff
@@ -40,7 +40,7 @@ To get a local copy up and running follw these simple example steps:
     - $ After the node_module module is installed successfully, run npm start
     - $ The project will open on the browser with localhost:800
 ```
-- Open the project in your favorite code editor
+Open the project in your favorite code editor
 
 ## Author
 
```

Comment:
- [ ] Please it will be nice to add license section to your readme file as this will make your readme file more informative and professional, checkout this [template](https://github.com/microverseinc/readme-template) from Microverse, it will guide you.

Reply:
OK, I followed the template to include a license section. Thank you!!!